design/templates.go

```diff
@@ -0,0 +1,46 @@
+// nolint: gochecknoglobals
+package design
+
+import (
+	_ "goa.design/goa/v3/codegen/generator"
+	. "goa.design/goa/v3/dsl"
+)
+
+var _ = API("templates", func() {
+	Title("Templates Service")
+	Description("A service for applying templates to Keboola projects")
+	Server("templates", func() {
+		Host("localhost", func() {
+			URI("http://localhost:8000")
+		})
+	})
+})
+
+var index = ResultType("application/vnd.templates.index", func() {
+	Description("Index of the service")
+	TypeName("Index")
+
+	Attributes(func() {
+		Field(1, "api")
+		Field(2, "documentation")
+	})
```

Comment:
Tieto attributes su `required`, tj. vzdy ich budeme vracat.
To ti fixne zbytocne `*string` (aby bola mozna `nil` hodnota)  vo vygenerovanom kode.

Reply:
+ hodil by sa aj `Example`

Reply:
![image](https://user-images.githubusercontent.com/19371734/156005573-c01da9f1-c1b2-416f-b34d-689ae99d778b.png)